lib/stripe/checkout/session.ex

```diff
@@ -47,9 +47,9 @@ defmodule Stripe.Session do
 
   @type subscription_data :: %{
           :items => list(item),
-          :metadata => Stripe.Types.metadata(),
-          :trial_end => integer(),
-          :trial_period_days => integer()
+          optional(:metadata) => Stripe.Types.metadata(),
+          optional(:trial_end) => integer(),
+          optional(:trial_period_days) => integer()
```

Comment:
looks like we have 3 new keys if you felt like adding!  I'll merge in a few days if you haven't.

`application_fee_percent`
`coupon`
`default_tax_rates`
`trail_from_plan`

Reply:
I can look at doing that, sure!

Reply:
I've added the new fields. I don't now if that is all that is necessary? I guess the system just handles the new information without having to add additional code?